app/forms/mappings/Formatters.scala

```diff
@@ -186,11 +247,16 @@ trait Formatters extends Transforms with Constraints {
 
   private[mappings] def bigDecimalTotalFormatter(itemsToTotal: String*): Formatter[BigDecimal] =
     new Formatter[BigDecimal] {
-
       override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], BigDecimal] = {
-        val total = itemsToTotal.foldLeft[BigDecimal](BigDecimal(0)) { (acc, next) =>
-          Try(BigDecimal(data.getOrElse(next, "0"))).toOption.fold(BigDecimal(0))(_ + acc)
-        }
+        val total =
+          itemsToTotal.foldLeft[BigDecimal](BigDecimal(0)) { (acc, next) =>
+            data.get(next) match {
+              case Some(str) =>
+                Try(BigDecimal(str)).getOrElse(BigDecimal(0.00)) + acc
+              case None =>
+                BigDecimal(0.00) + acc
```

Comment:
BigDecimal(0.00) is redundant here - you can just remove it.

Reply:
Done